### ISSUE TITLE
PoolV1.getExpectedOutputs() misleading docs

### ISSUE THREAD

**Issue by Ludwintor** [ston-fi/sdk#43]
This method has following docstring
https://github.com/ston-fi/sdk/blob/34a6751a77e0e9cff67542bed668610500705650/src/contracts/dex/v1/PoolV1.ts#L153-L167
Line about `params.jettonWallet` says that it must be equal to one of the jetton addresses of the **POOL**

Get two jetton wallets of this pool pair and use them with this method - everytime get zeros.
Get two jetton wallets of ROUTERv1 and use them with this method - get expected values from both wallets.

So these must be jetton addresses of the ROUTER, not pool?
Tested it only on PoolV1

**Comment by pavel-rakhmanov** [comment]
Yep, you are right. `jettonWallet` here should be owned by the Router, not a Pool. Thank you for finding this issue; I will update the comment in the next release.

By the way, the `getExpectedOutputs` method only exists in v1 contracts and does not have an analog in v2 contracts. You can use our API to simulate swaps
- https://api.ston.fi/swagger-ui/#/Dex/dex_simulate_swap
- https://api.ston.fi/swagger-ui/#/Dex/dex_reverse_simulate_swap
